app/src/main/java/de/danoeh/antennapod/fragment/swipeactions/SwipeActions.java

```diff
@@ -261,4 +236,23 @@ public int getMovementFlags(@NonNull RecyclerView recyclerView, @NonNull Recycle
     public void startDrag(EpisodeItemViewHolder holder) {
         itemTouchHelper.startDrag(holder);
     }
+
+    public static class Actions {
+        public SwipeAction right = null;
```

Comment:
final, @NonNull?

Reply:
right or left could be null but that is always checked by `hasActions` before its accesed or what are you reffering to?
The Actions itself wont be null as its init in the SwipeActions constructor